config/initializers/acts_as_taggable_on.rb

```diff
@@ -0,0 +1,2 @@
+ActsAsTaggableOn.delimiter = ' '
```

Comment:
let's keep the coma as the default delimiter, that way it forces us to use one word tags